core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -2699,6 +2699,35 @@ private void checkPushJoinThroughUnionOnRightDoesNotMatchSemiOrAntiJoin(JoinRelT
         .check();
   }
 
+  /** Tests {@link org.apache.calcite.rel.rules.MinusToDistinctRule},
+   * which rewrites an {@link Minus} operator with 3 inputs. */
+  @Test void testMinusToDistinct() {
+    final String sql = "select EMPNO,ENAME,JOB from emp where deptno = 10\n"
+        + "except\n"
+        + "select EMPNO,ENAME,JOB from emp where deptno = 20\n"
+        + "except\n"
+        + "select EMPNO,ENAME,JOB from emp where deptno = 30\n";
+    sql(sql)
+        .withRule(CoreRules.MINUS_MERGE,
+            CoreRules.MINUS_TO_DISTINCT)
+        .check();
+  }
+
+  /** Tests {@link org.apache.calcite.rel.rules.MinusToDistinctRule},
+   *  correctly ignores an {@code EXCEPT ALL}. It can only handle
+   *    * {@code EXCEPT DISTINCT}.*/
```

Comment:
Is this comment formatted incorrectly? Looks like there is an extra * and could use a space at the end